managers.md

```diff
@@ -1,115 +1,107 @@
+through managing a manager.
+
+### Management & Leadership
+
+- You manage up to two teams either directly or through managing a manager within the same Organization.
+- You manage & coach ICs. You look for opportunities to grow or manage new engineering managers (EMs) or Staff Engineers. You have multiple instances of successful promos.
+- You plan for team needs through suggesting headcount, proposing tools & integrations, and identifying dependencies. You understand what needs other teams within the same Organization have, and escalate or adjust if those needs will impede your team’s needs. You understand the strategy for the Organization, and ensure your team is lined up to support it.
+- You scout and attract talent. You build relationships with potential candidates. You influence org wide hiring and staffing. You moderate hiring decisions.
+- You identify macro-level patterns of team engagement, influence decisions on retention.
 
 ### Software Engineering & Design
 
-- You hold your team to the software engineering and design standards we have set for individual contributors.
-- You provide an environment (process, tools, goals, sprints, etc) that allows individual contributors to do their best work.
-- When you contribute technically, you exemplify the standards we set for individual contributors.
+- You build technical experts within the team who can continue building out best practices (e.g. tech leads). You ensure these technical experts have opportunities to help the team build technical skills.
+- You are a technical and interpersonal role model. You help resolve technical issues and advise on technical decisions.
 
 ### Execution & Results
 
-- You have established a track record of effectively managing your time and know when to ask for help to unblock yourself. When blocked, you are resourceful and take initiative in finding other tasks to complete.
-- You help create organization and structure around large projects & problems that aren’t well defined, thereby setting your team up to successfully deliver on initiatives.
-- You have established a track record of investigating and vetting the reuse of existing components and services when approaching new projects and feature requests.
-- You can orchestrate and coordinate projects & solve problems that span across multiple teams.
-- You are responsible for working with stakeholders (PM, Design, DS, etc) to drive planning for your teams OKRs and goals.
-- You partner with stakeholders (PM, Design, DS, etc) to prioritize quarterly and sprint goals as well as allocate work accordingly and effectively.
-- You provide timely code reviews and are responsive to peer feedback on code reviews and learn from it.
+- You delegate team process ownership to members of the team, load balancing as needed. You identify areas that need improvement and solicit owners from the team. You share team processes across the same Organization.
```

Comment:
Just noting the new competencies don't mention anything about established track records, and I find that a good distinction between similar competencies across different levels

Reply:
I agree that many times a "senior" in a role is largely characterized by having more experience in the base responsibilities.  My hope is that this is a generality that doesn't need to be pointed out for each common responsibility, but we can focus on the differentiating factors to keep these competencies simple and understandable.  That being said, let's resurface this conversation when we decide to update the EM and Director competencies to the new form to see if there are any areas that could use this clarifier.